www/src/styles/algolia/style.css

```diff
@@ -63,6 +63,7 @@ html.dark {
 }
 .DocSearch-Search-Icon {
   stroke-width: 1.6;
+  margin-right: 15px;
```

Comment:
```suggestion
  margin-inline-end: 15px;
```

This doesn't work in RTL languages as it places the margin on the wrong side. Could be fixed by using inline direction instead.

<img width="582" alt="image" src="https://user-images.githubusercontent.com/8353666/208951363-6c910de5-8d5b-482d-bf4f-9af78c47298e.png">

Reply:
Thanks for the review. 

Oh yes, I didn't think about the RTL languages, thanks for your suggestion.

All of the mentioned issues should be fixed with the latest changes.